files/justfiles/secureblue.just

```diff
@@ -79,6 +79,12 @@ remove-kargs-hardening:
       --delete-if-present="ia32_emulation=0"
     echo "Hardening kargs removed."
 
+# Ensure the signed image is being used
+switch-to-signed:
```

Comment:
this is no longer needed as of https://github.com/secureblue/secureblue/pull/564/files#diff-d48867bc4fd1881c03a2b2fee68a985512e7633905af8ea05fd5f62ccb8021ee